RULES.md

```diff
@@ -223,5 +223,118 @@ your code.
 * Always use semicolons.
 
   ```js
+<<<<<<< HEAD
```

Comment:
Fixed, thanks again.  This time I did a grep for these and they should all be resolved :)